### ISSUE TITLE
Chapters in MangaSearchEntry can be null

### ISSUE THREAD

**Issue by moiph** [Ervie/jikan.net#15]
Sample request: `https://api.jikan.moe/v3/search/manga?q=naruto`
```
...
"results":[{"mal_id":95210,"url":"https:\/\/myanimelist.net\/manga\/95210\/Boruto__Naruto_Next_Generations","title":"Boruto: Naruto Next Generations","image_url":"https:\/\/cdn.myanimelist.net\/images\/manga\/3\/181968.jpg","type":"Manga","volumes":null,"chapters":null,"publishing":true,
...
```

```
JikanDotNet.Exceptions.JikanRequestException: Serialization failed.
Inner exception message: Error converting value {null} to type 'System.Int32'. Path 'results[0].chapters', line 1, position 404.
 ---> Newtonsoft.Json.JsonSerializationException: Error converting value {null} to type 'System.Int32'. Path 'results[0].chapters', line 1, position 404.
 ---> System.InvalidCastException: Null object cannot be converted to a value type.
   at System.Convert.ChangeType(Object value, Type conversionType, IFormatProvider provider)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.EnsureType(JsonReader reader, Object value, CultureInfo culture, JsonContract contract, Type targetType)
   --- End of inner exception stack trace ---
```

**Comment by Ervie** [comment]
This is kinda weird, chapters and volumes should default themselves to 0 when there is no definite number - this is the result I got when calling endpoint mentioned in the sample. I guess this might be something related to caching changes.

I guess changing this to nullable property (the same for volumes and episodes in anime search) won't hurt.

**Comment by Ervie** [comment]
Done, it is patched in version 1.4.1.
